src/debugger/scriptImporter.ts

```diff
@@ -82,8 +84,7 @@ export class ScriptImporter {
     }
 
     public downloadDebuggerWorker(sourcesStoragePath: string, projectRootPath: string): Q.Promise<void> {
-        const errPackagerNotRunning = new RangeError(`Cannot attach to packager. Are you sure there is a packager and it is running in the port ${this.packagerPort}? If your packager is configured to run in another port make sure to add that to the setting.json.`);
```

Comment:
That's how this error would handle after this change
![image](https://user-images.githubusercontent.com/32372875/50150125-1fb53a80-02ce-11e9-9c12-ee0a6e6f4718.png)